playbooks/compute/nto/roles/configurecluster/tasks/ppc.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Create performance Profile
+  ansible.builtin.include_tasks: create_directory.yml
+  vars:
+    dir_path: "{{ must_gather_dir }}"
+
+- name: Collect Must Gather
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  ansible.builtin.shell: |
+    oc adm must-gather --dest-dir={{ must_gather_dir }}
+  register: must_gather
+  changed_when: must_gather.rc != 0
+
+- name: Save must gather to artifacts
+  community.general.archive:
+    path: "{{ must_gather_dir }}"
+    dest: "{{ artifacts_folder }}/pre-mustgather.tar.gz"
+    mode: 775
+    format: gz
+
+- name: Get Image SHA
```

Comment:
Same here with the register name, give it a more easy to understand name

Reply:
thanks - changed